File: Programmers/Lv.1/유연근무제/388351.java
class Solution {
    public int solution(int[] schedules, int[][] timelogs, int startday) {
        int answer = timelogs.length;
        
        for (int i = 0; i < timelogs.length; i++) {
            for (int j = 0, start = startday; j < timelogs[i].length; j++, start++) {
                if (start % 7 == 0 || start % 7 == 6) continue;
                
                int expired = schedules[i] + 10;
                if (expired % 100 >= 60) expired += 40;
                
                if (timelogs[i][j] > expired) {
                    answer--;
                    break;
                }
            }
        }
        
        return answer;
    }
}

File: Programmers/Lv.1/유연근무제/388351.js
function solution(schedules, timelogs, startday) {
    let answer = timelogs.length;
    
    for (let i = 0; i < timelogs.length; i++) {
        for (let j = 0, start = startday; j < timelogs[i].length; j++, start++) {
            if (start % 7 === 0 || start % 7 === 6) continue;
            
            let expired = schedules[i] + 10;
            if (expired % 100 >= 60) expired += 40;
            
            if (timelogs[i][j] > expired) {
                answer--;
                break;
            }
        }
    }
    
    return answer;
}
